src/cdp/cdp.zig

```diff
@@ -54,6 +54,9 @@ pub fn CDPT(comptime TypeProvider: type) type {
         // The active browser
         browser: Browser,
 
+        // when true, any target creation must be attached.
+        target_auto_attach: bool = false,
```

Comment:
@karlseguin I'm not this is the right place for the setting.
I think it's a state relative to the whole cdp session, not the browser's context.
WDYT?

Reply:
The documentation says it enables/disables for targets 'which are related to this one'. Given we have a dummy "browser" target, and a single BrowserContext / target at a time, it's probably safe to assume they're all "related" to each other.

Reply:
Hum, good point :thinking:
Maybe it's better to move to to the browser's context instead?
and fail or create one automatically if no bc exists :thinking:

Reply:
I think we can try to leave it here globally and see if it causes any issues. It feels like most clients just try to enable it for _everything_, and I haven't noticed anything try to disable it, so having it global should work.